.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Found a problem? Help us improve.
+title: "[BUG] Title"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+Notes:
+- Please do not make support requests on Github. Our issue tracker is for tracking bugs and feature requests only. If you need help configuring the emulator please make a [request on our forums](https://forums.pcsx2.net/index.php) or contact us on [discord](https://discord.com/invite/TCz3t9k). If you are unsure, start with discord or the forums.
+- We only accept bug reports for the latest dev version. Please try upgrading before making an issue.
+- Please make an effort to make sure your issue isn't already reported.
+- We are not accepting bug reports for PSX mode at this time. If you are interested in helping contribute to PSX mode please do so on the [forums](https://forums.pcsx2.net/index.php). Otherwise our recommendation is that you use a proper [PSX emulator](https://emulation.gametechwiki.com/index.php/PlayStation_emulators).
```

Comment:
Generally my issue with templates is always that they put a pile of placeholder text that ends up on the final issue and makes it harder to read.  I'd lean towards putting any exposition in comments so it doesn't end up on the final issue regardless of whether-or-not the creator cleans things up. For example - https://github.com/aws/aws-sdk-java/issues/new?labels=bug%2C+needs-triage&template=bug-report.md

Reply:
Good call.